src/components/data_comps/datm/cime_config/config_component.xml

```diff
@@ -59,22 +59,15 @@
   <entry id="DATM_PRESAERO">
     <type>char</type>
     <valid_values>none,clim_1850,clim_2000,trans_1850-2000,rcp2.6,rcp4.5,rcp6.0,rcp8.5,cplhist</valid_values>
-    <default_value>none</default_value>
+    <default_value>clim_2000</default_value>
```

Comment:
Just to confirm: You have removed 'none' as ever appearing as a default, but left it in the list of valid values. Is it correct that 'none' is still a valid value?

Reply:
Yes - that is correct.